crates/ruff_linter/src/rules/flake8_debugger/rules/debugger.rs

```diff
@@ -109,6 +109,8 @@ fn is_debugger_call(call_path: &CallPath) -> bool {
             ]
             | ["celery", "contrib", "rdb", "set_trace"]
             | ["builtins" | "", "breakpoint"]
+            | ["debugpy", "breakpoint" | "listen" | "wait_for_client"]
+            | ["ptvsd", "break_into_debugger" | "wait_for_attach"]
```

Comment:
I tried to pick the API methods that looked like they would be most commonly used based on the documentation:
https://github.com/microsoft/debugpy/?tab=readme-ov-file#debugpy-import-usage
https://github.com/microsoft/ptvsd?tab=readme-ov-file#ptvsd-import-usage

Reply:
Spot-checked, makes sense to me.